code/run_iLasso_row.m

```diff
@@ -4,24 +4,26 @@
 p1=params.p1;
 std_dev=params.kernel_width;
 [LX,WX] = size(m,'X');
-numrows = LX;
 if ismember('regix',who(m))
-    numregs = length(m,'regix');
+    regix = m.regix;
```

Comment:
Currently we assume that the user has already mapped regulator names to row indices so that this information is available in the input `mat` file.